travisci/python-linter_harness.sh

```diff
@@ -15,29 +15,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PYTHON_SOURCE_LOCATIONS=('scripts' 'src/python')
-
 # Setup the environment variables
 # shellcheck disable=SC2155
 export PYTHONPATH=$PYTHONPATH:$(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
 # more info: https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules
 export MYPYPATH=$MYPYPATH:src/python/lib
 
-PYLINT_OUTPUT_FILE=$(mktemp)
-PYLINT_ERRORS=$(mktemp)
-# CITest project is on hold and it needs to be updated before resuming its linter checker
-find "${PYTHON_SOURCE_LOCATIONS[@]}" -type f -name "*.py" \! -name "Ortheus.py" \! -name "*citest*.py" \! -path "*/citest/*" -print0 | xargs -0 pylint --rcfile pyproject.toml --verbose | tee "$PYLINT_OUTPUT_FILE"
-grep -v "\-\-\-\-\-\-\-\-\-" "$PYLINT_OUTPUT_FILE" | grep -v "Your code has been rated" | grep -v "\n\n" | sed '/^$/d' > "$PYLINT_ERRORS"
-! [ -s "$PYLINT_ERRORS" ]
+# Function to run pylint
+run_pylint() {
+  local pylint_output_file=$(mktemp)
+  local pylint_errors=$(mktemp)
```

Comment:
Thanks for taking on board my feedback, @thiagogenez.

It looks like there's no need for the `$pylint_errors` file now:
```suggestion
```

Reply:
Whoops! It looks like this suggestion is already out of date.

Reply:
Yeah - I have just removed it!